Homework17/Tests/UsersTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Homework17.Helpers;
+using Homework17.Models.JsonModels;
+using Homework17.Services;
+using NUnit.Framework;
+using System.Net;
+
+namespace Homework17.Tests
+{
+    public class UsersTests
+    {
+        private const string DateTimeFormat = "yyyy-MM-dd HH:mm";
+
+        // Test 1: Get list of users and check the number of users.
+        [Test]
+        public void CheckNumberOfUsers()
+        {
+            const int expectedTotalUsers = 12;
+
+            var response = RequestsService.GetUsersList();
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual(HttpStatusCode.OK, response.StatusCode);
```

Comment:
First 2 asserts absolutely the same. You can leave only one

Reply:
The same below

Reply:
yep, it was just to show that i know both ))